src/app/frontend/common/components/textinput/component.ts

```diff
@@ -68,7 +69,13 @@ export class TextInputComponent implements OnInit, AfterViewInit, OnChanges {
   constructor(private readonly themeService_: ThemeService) {}
 
   ngOnInit(): void {
-    this.theme = this.themeService_.isLightThemeEnabled() ? EditorTheme.light : EditorTheme.dark;
+    switch (this.themeService_.theme) {
```

Comment:
In the next change this will be switched to just checking if the theme is dark or light. It's planned in https://github.com/kubernetes/dashboard/issues/5751.

Reply:
Done.